pkg/webhook/parser.go

```diff
@@ -38,8 +38,11 @@ import (
 
 // The default {Mutating,Validating}WebhookConfiguration version to generate.
 const (
-	v1                    = "v1"
-	defaultWebhookVersion = v1
+	v1                      = "v1"
+	defaultWebhookVersion   = v1
+	defaultServiceName      = "webhook-service"
+	defaultServiceNamespace = "system"
+	defaultServicePort      = 443
```

Comment:
```suggestion
```
unused (I assume because we don't want to change the default behavior)

Reply:
Removed. I was going to [use it here](https://github.com/davidxia/controller-tools/blob/0f0cdbcf92ba6133c860696bd2da781722416d3f/pkg/webhook/parser.go#L346-L348) but then saw in the [K8s docs](https://v1-30.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.30/#servicereference-v1-admissionregistration-k8s-io)

> Default to 443 for backward compatibility.

Do you think a nil-pointer for `port` will default to 443 for backwards compat?

Reply:
ah yes, that's the expected behavior and what's happening with the code right now